dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/test-bundles/io.openliberty.opentracing.spi.tracer/src/io/openliberty/opentracing/spi/tracer/OpentracingTracerFactory.java

```diff
@@ -0,0 +1,27 @@
+/*****************************************************************************
+ * Copyright (c) 2017, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *****************************************************************************/
+
+package io.openliberty.opentracing.spi.tracer;
+
+import io.opentracing.Tracer;
+
+/**
+ *
+ */
+public interface OpentracingTracerFactory {
+
```

Comment:
I thought this code was included in the the user feature jar. Why do you also have the source here?